pkg/moov/bank_account_api.go

```diff
@@ -32,7 +32,7 @@ func WithMX(mx MXRequest) CreateBankAccountType {
 	})
 }
 
-func WaitForPaymentMethod() callArg {
+func WaitForPaymentMethod() CreateBankAccountType {
```

Comment:
Wait, should this be something else?

Reply:
Good point. `X-Wait-For: payment-method` is used for [card linking](https://github.com/search?q=repo%3Amoovfinancial%2Fplatform-api%20%22payment-method%22&type=code) as well, so we should allow it on both endpoints.

Reply:
We currently hardcode that on CreateCard though, so it can't be supplied on the request.

https://github.com/moovfinancial/moov-go/blob/main/pkg/moov/cards.go#L77

Reply:
ah i see